draft-ietf-quic-http.md

```diff
@@ -147,11 +149,16 @@ Origins SHOULD list only versions which are supported by the alternative, but
 MAY omit supported versions for any reason.
 
 
-# Connection Establishment {#connection-establishment}
+## Connection Establishment {#connection-establishment}
+
+HTTP/QUIC relies on QUIC as the underlying transport.  The QUIC version being
+used MUST use TLS version 1.3 or greater as its handshake protocol.  The Server
+Name Indication extension {{!RFC6066}} MUST be included in the TLS handshake.
 
-HTTP/QUIC connections are established as described in {{QUIC-TRANSPORT}}. During
+QUIC connections are established as described in {{QUIC-TRANSPORT}}. During
 connection establishment, HTTP/QUIC support is indicated by selecting the ALPN
-token "hq" in the crypto handshake.
+token "hq" in the crypto handshake.  Support for other application-layer
```

Comment:
nit: s/crypto/cryptographic to better align with Transport doc terminology. Seems like an opportunity to fix it up while in this area.

Reply:
How about "TLS," since I've already said we're bound to that anyway?  ;-)